routes/editor.js

```diff
@@ -422,6 +430,9 @@ router.post('/page/imagebackground/id/:id', function (req, res) {
         // format and save new item
         var fullpage = (newItem['fullpage'] === 'on') ? true : false;
         item['format'] = { fullpage: fullpage };
+        var suppress = (newItem['suppress'] === 'on') ? true : false;
+        item['in_nav'] = suppress;
```

Comment:
let's make in_nav true if it should be in the nav...

Reply:
It's on by default so it's more the fact that it should be reversed. I've changed `in_nav` to just `suppress` which although means the word loses all meaning after the amount of times I've seen it but at least it has an inner logic which is more obvious to an outsider.